coordinator/cql/src/main/java/io/stargate/cql/impl/CqlImpl.java

```diff
@@ -79,7 +80,9 @@ public void start() {
     int nativePort = TransportDescriptor.getNativeTransportPort();
     int nativePortSSL = TransportDescriptor.getNativeTransportPortSSL();
     InetAddress nativeAddr = TransportDescriptor.getRpcAddress();
-
+    // Cassandra {4.0.10}
```

Comment:
Quick questions on this:

1. What is the scope of copied code: this is where copied part starts but is it just next statement, or longer block
2. Compared to non-annotated code, how are these different? As in, was other code copied earlier (before this PR), or something that did not necessarily come from C* codebase?